src/TrustedShops.php

```diff
@@ -225,10 +225,19 @@ public static function wp_footer() {
 
     $products_sku = [];
     if ($product->get_type() === 'variable') {
-      $variations = $product->get_available_variations();
-      foreach ($variations as $variation) {
-        $products_sku[] = $variation['sku'] ?? '';
-      }
+      // Gets variations skus, avoiding "get_available_variations()".
+      global $wpdb;
```

Comment:
Globals should be imported only once at the beginning of the function - please move this accordingly.